### PR TITLE
set job default status as submitted until get some result

### PR DESCRIPTION
**Pull request by SuperYoko**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 

https://github.com/vesoft-inc/nebula-ent/issues/1295

## How do you solve it?

set default to a new status as E_JOB_SUBMITTED.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/graph/executor/admin/SubmitJobExecutor.cpp [hunk 1]
```diff
@@ -151,7 +151,7 @@ nebula::DataSet SubmitJobExecutor::buildShowResultData(
                           apache::thrift::util::enumNameSafe(tsk.get_result()),
                           convertJobTimestampToDateTime(std::move(tsk).get_start_time()),
                           convertJobTimestampToDateTime(std::move(tsk).get_stop_time()),
-                          jd.get_code() == nebula::cpp2::ErrorCode::E_UNKNOWN
+                          jd.get_code() == nebula::cpp2::ErrorCode::E_JOB_PROCESSING
```
Comment (panda-sheep) [inline src/graph/executor/admin/SubmitJobExecutor.cpp:154]: Remember to add `E_JOB_PROCESSING` to common.thrift and `MetaClient::handleResponse`

File: src/meta/processors/job/JobDescription.h [hunk 2]
```diff
@@ -176,7 +176,7 @@ class JobDescription {
   Status status_;
   int64_t startTime_;
   int64_t stopTime_;
-  nebula::cpp2::ErrorCode errCode_{nebula::cpp2::ErrorCode::E_UNKNOWN};
+  nebula::cpp2::ErrorCode errCode_{nebula::cpp2::ErrorCode::E_JOB_PROCESSING};
```
Comment (panda-sheep) [inline src/meta/processors/job/JobDescription.h:179]: The default is `E_JOB_PROCESSING`, which doesn't feel good

Comment (SuperYoko) [reply src/meta/processors/job/JobDescription.h:179]: > The default is `E_JOB_PROCESSING`, which doesn't feel good

Yes,  I know.
Wait for me to think of an awesome name. XDDD

File: src/graph/executor/admin/SubmitJobExecutor.cpp (hunk 1 above)
Comment (SuperYoko) [reply src/graph/executor/admin/SubmitJobExecutor.cpp:154]: Thanks.

File: src/meta/processors/job/JobDescription.h (hunk 2 above)
Comment (SuperYoko) [reply src/meta/processors/job/JobDescription.h:179]: What about E_SUBMITTED?

Comment (panda-sheep) [reply src/meta/processors/job/JobDescription.h:179]: What about the display is empty when the job status is `nebula::cpp2::ErrorCode::E_UNKNOWN`？

Comment (SuperYoko) [reply src/meta/processors/job/JobDescription.h:182]: There's some condition job return an unknown error ...

**Comment by critical27** [comment]
set error code to empty?

**Comment by SuperYoko** [comment]
> How about set error code to empty? Is it because some storage will report E_UNKNOWN?

Yes, for jobmanager there's E_UNKNOWN error.

**Review by critical27 — DISMISSED** [review]

**Review by liwenhui-soul — APPROVED** [review]

**Review by pengweisong — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Comment by jinyingsunny** [comment]
checked
image

**Comment by jinyingsunny** [comment]
image
3.3.0-ent

**Pull request merged**